fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/DruidPartialDataResponseProcessor.java

```diff
@@ -131,9 +132,14 @@ public void processResponse(JsonNode json, DruidAggregationQuery<?> query, Loggi
             } else {
                 next.processResponse(json.get(DruidJsonResponseContentKeys.RESPONSE.getName()), query, metadata);
             }
+        } else if (statusCode == Status.NOT_MODIFIED.getStatusCode() && next instanceof FullResponseProcessor) {
```

Comment:
if 304, and the next response Processor is FullResponseProcessor (which can be etagCacheResponseProcessor), we throw an error?